cli/cmd/summary.go

```diff
@@ -62,3 +73,87 @@ func checkFilePath(filename string) error {
 	dirPath := filepath.Dir(filename)
 	return os.MkdirAll(dirPath, 0755)
 }
+
+func generateSummaryTable() *schema.Table {
+	tableName := "cloudquery_sync_summaries"
+	t := schema.Tables{{
+		Name: tableName,
+		Transform: transformers.TransformWithStruct(
+			&syncSummary{},
+		),
+		Columns: []schema.Column{},
+	}}
+	if err := transformers.TransformTables(t); err != nil {
+		panic(err)
```

Comment:
Can we return an error instead of panicking?

Reply:
Done in [960a887](https://github.com/cloudquery/cloudquery/pull/17842/commits/960a8873f08ee5e4d4586adf1d648b57ae23d58c)